src/diffusers/loaders/single_file.py

```diff
@@ -309,7 +310,7 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
                 init_kwargs.update(components)
 
         additional_components = set_additional_components(
-            class_name, original_config, checkpoint=checkpoint, model_type=model_type
+            class_name, original_config, checkpoint=checkpoint, model_type=model_type, is_cosxl_edit=is_cosxl_edit
```

Comment:
Not needed. 
https://github.com/huggingface/diffusers/blob/08bf7545070764d41637e90290ca9b92b392263e/src/diffusers/loaders/single_file.py#L317